lib/solidus_feeds.rb

```diff
@@ -5,3 +5,6 @@
 
 require 'solidus_feeds/version'
 require 'solidus_feeds/engine'
+
+generators = File.join(__dir__, 'solidus_feeds/generators/*.rb')
+Dir[generators].each { |file| require file }
```

Comment:
just a nit:

```suggestion
Dir[generators].sort.each { |file| require file }
```

This would ensure the loading order is always the same